README.md

```diff
@@ -142,19 +150,20 @@ rainforest report <run-id> --junit-file rainforest.xml
 
 Upload a CSV to create a new tabular variables.
 ```bash
-rainforest csv-upload --import-variable-csv-file PATH/TO/CSV.csv --import-variable-name my_variable
+rainforest csv-upload PATH/TO/CSV.csv --import-variable-name my_variable
```

Comment:
like above `rainforest csv-upload --import-variable-name my_variable PATH/TO/CSV.csv` should be better and more in line with how it's generally used, although both should work.